### ISSUE TITLE
Wild cards in non-positional rules

### ISSUE THREAD

**Issue by daviortega** [biowonks/projects#13]
If the non-positional rule uses wild cards, it currently does not work.

For example, the rule
```javascript
Npos: [{
    resource: 'pfam28',
    feature: '.*',
    count: '{1}'
}]
```
does not select any sequence from the test suit.

So it would be simple to just forbidden the use of wild cards in non-positional rules because the effect can be pretty much achieved with positional rules. However, we use this as a *hack* to increase the scope of resources for the use of wild-cards in features with positional rules. For example:

*Filter protein sequences starting with 1 TM follow by any 2 features (from pfam28 or TM) and another TM*
```javascript
[
	{
		pos: [
			{
				resource: 'fql',
				feature: '^'
			},
			{
				resource: 'das',
				feature: 'TM',
				count: '{1}'
			},
			{
				resource: '.*',
				feature: '.*',
				count: '{2}'
			},
			{
				resource: 'das',
				feature: 'TM',
				count: '{1,}'
			}
		],
		Npos: [
			{
				resource: 'pfam28',
				feature: '.*', // <--------------------------- HERE
				count: '{0,}'
			}
		]
	}
]
```
It seems that the right thing to do is to fix the wild-card use in non-positional rules.

**Comment by daviortega** [comment]
Found the problem. The mechanism that I use to check the non-positional rule is using `.split()` and the wild card makes it impossible to get it. Instead of searching for `@pfam28`, it tries to split with `.*pfam28` which does not exists.

Solution: pass `''` from feature if `.*` is passed in non-positional rules for the sake of coherence with positional rules. This should be happening in the rule parsing step.

**Comment by daviortega** [comment]
Fixed in cb0dba713db7d59bf81878cea1653fea24a66b38
